src/components/lyne-cta-button/lyne-cta-button.scss

```diff
@@ -1,6 +1,21 @@
 :host {
```

Comment:
I discussed it with Mänu. I told him, that in my opinion:
- truncation is almost never a good think
- we must have content guidelines that advice authors, designers and developers to use short expressions for buttons to prevent it from going multiline
- if there is too much text in the button label, it is not the fault of the end-user, so we should not punish the end-user with truncation ;)

He was convinced by my arguments and accepted, that - in edge cases - the button can go multiline.